pages/apis/rest_api/annotations.md

```diff
@@ -51,3 +51,77 @@ curl "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines/{pipeline.
 Required scope: `read_builds`
 
 Success response: `200 OK`
+
+## Create an annotation on a build
+
+Creates an annotation on a build.
+
+>📘
+> Note that you need the <a href="/docs/apis/rest-api/builds#build-number-vs-build-id">build number</a> to create annotations, not the build ID.
+
+```bash
+curl -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines/{pipeline.slug}/builds/{build.number}/annotations" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "body": "Hello world!",
+    "style": "info",
+    "context": "greetings"
+  }'
+```
+
+```json
+{
+  "id": "018b8d10-6b5b-4df2-b0ff-dfa2af566050",
+  "context": "greeting",
```

Comment:
Nitpick: The request has `greetings` as the `context` but the response drops the `s`. I assume these should match?

```suggestion
  "context": "greetings",
```

^ or the other way around.